.github/workflows/monitor-components.yml

```diff
@@ -53,6 +53,7 @@ jobs:
             feed: https://github.com/curl/curl/tags.atom
           - label: gpg
             feed: https://github.com/gpg/gnupg/releases.atom
+            title-pattern: ^gnupg-[0-9]+\.[0-9]*[02468]\.
```

Comment:
I'd change this to `^gnupg-2\.3\.` for now.
This doesn't seem to be as fixed of a rule as your proposed pattern implies. Judging by the time between 2.1.0 and 2.2.0 as well as 2.2.0 and 2.3.0 it'll probably be 2024 before we see GPG 2.4.0 anyways. If there are strong hints of 2.3 becoming stable we might even want to drop the pattern before 2.4.0 gets released.

Reply:
Good point. Let me change the code.

Reply:
(Or not, but abandon the PR instead ;-))